apps/client/src/app/pages/portfolio/fire/fire-page.component.ts

```diff
@@ -109,6 +109,24 @@ export class FirePageComponent implements OnDestroy, OnInit {
       });
   }
 
+  public onTargetNetWorthChanged(targetNetWorth: number) {
```

Comment:
Rename accordingly

Reply:
Changed to projectedTotalAmount in latest commit